### PR TITLE
Feature/86cuw4czn Services Page

### PR DESCRIPTION
**Pull request by NikkiShao**
Completed the whole services page including displaying service cards, routing to particular services pages (this is a separate page not covered), pagination of items, etc. The search bar functionality is not covered by this feature.

To test the page with database data, tester/reviewer may need to import some data to their local MongoDB server. Instructions have also been included in the mockdata folder.

### CODE REVIEW

File: behind-the-veil-siteroot/imports/api/collections/photo.js
Comment (nethbtju) [inline behind-the-veil-siteroot/imports/api/collections/photo.js:1]: is this suppose to empty?

**Comment by NikkiShao** [comment]
Please squash all these into 1 commit when merging

File: behind-the-veil-siteroot/imports/api/publications/photo.js
Comment (nethbtju) [inline behind-the-veil-siteroot/imports/api/publications/photo.js:1]: Delete!!

File: behind-the-veil-siteroot/imports/ui/App.jsx
Comment (nethbtju) [inline behind-the-veil-siteroot/imports/ui/App.jsx:1]: This is weird, it should've have nothing I just merged anushas?

File: behind-the-veil-siteroot/imports/ui/components/pages/login/LoginPage.jsx
Comment (nethbtju) [inline behind-the-veil-siteroot/imports/ui/components/pages/login/LoginPage.jsx:1]: do we wanna change the naming convention of the page to sign in page ?

Comment (nethbtju) [reply behind-the-veil-siteroot/imports/ui/components/pages/login/LoginPage.jsx:1]: Also this is so weird its as if Kyles stuff doesn't exist on develop

**Review by nethbtju — CHANGES_REQUESTED** [review]
Couple of changes

**Comment by nethbtju** [comment]
Yours seems to have the duplicate commits issue too

**Review by nethbtju — APPROVED** [review]
Looks fine now!

**Review by nethbtju — APPROVED** [review]
All files checked and reviewed no issues

**Review by nethbtju — APPROVED** [review]
Reviewed changes

**Review by joshualoongwy — APPROVED** [review]

**Pull request merged**